### PR TITLE
Update drivers instructions

### PR DESCRIPTION
**Pull request by katarinasupe**
### Description

Currently checking all driver codes. 

**Memgraph docs**
- [x] Python
- [x] Rust - add working non-Dockerized example
- [x] C# - v5
- [x] Java - v5
- [x] Go - v5
- [x] Node.js 
- [x] JavaScript
- [x] PHP

**Cloud docs**
- [x] Python
- [x]  Rust - update `rsmgclient` version and add new working code
- [x] Java - left as it is (v4) and added info about v5 (link to Memgraph docs)
- [x] C# - returned driver v4 and added info about v5 (link to Memgraph docs)
- [x] Go - updated to v5
- [x] PHP
- [x] Node.js - updated link to quick start guide

**Cloud (cloud.memgraph.com)** 
- [x] Python
- [x] Rust - update `rsmgclient` version and add new working codes
- [x] Java - left as it is 
- [x] C# - returned driver v4
- [x] Go
- [x] PHP
- [x] Node.js - updated link to quick start guide

### Pull request type

- [x] Documentation fix

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

**Review by kgolubic — APPROVED** [review]
Everything looks ok.

File: memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md [hunk 1]
```diff
@@ -77,24 +73,17 @@ public class HelloWorld implements AutoCloseable
...
-                            parameters( "message", message ) );
-                    return result.single().get( 0 ).asString();
-                }
-            } );
-            System.out.println( greeting );
-        }
+    	
+    	try (var session = driver.session()) {
+    		var hello = session.executeWrite( transaction -> {
+    		var query = new Query("CREATE (a:Greeting) " + "SET a.message = $message " + "RETURN 'Node ' + id(a) + ': ' + a.message", parameters("message", message));
```
Comment (gitbuda) [inline memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md:79]: you can remove all + here

Comment (katarinasupe) [reply memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md:79]: I don't understand what you mean :)

**Comment by vpavicic** [comment]
DO you think we should propagate the changes to previous versions?

File: memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md (hunk 1 above)
Comment (gitbuda) [reply memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md:79]: sorry... `"CREATE (a:Greeting) " + "SET a.message = $message "` can be `"CREATE (a:Greeting) SET a.message = $message "`

Comment (gitbuda) [reply memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md:79]: a saw this pattern twice

**Comment by katarinasupe** [comment]
> DO you think we should propagate the changes to previous versions?

I think there is no need @vpavicic, maybe even better we have both so the users have options with old drivers too.

File: memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md (hunk 1 above)
Comment (katarinasupe) [reply memgraph_versioned_docs/version-2.5.2/connect-to-memgraph/drivers/java.md:79]: Thanks, will fix it!

**Pull request merged**